### PR TITLE
EUI 3384 hyphens in case number

### PR DESCRIPTION
**Pull request by ishOsw**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-3384

### Change description ###

The banner displayed on the screen when a Probate case has been created does not include hyphens in the case number fix

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by leepavlouhmcts — APPROVED** [review]
looks good

**Review by Daniel-Lam — DISMISSED** [review]
@ishOsw - looks OK to me, but something to perhaps bear in mind for future... it would be good to extract the logic in here into a separate function, which is then called from `ngOnInit()`. Having a separate function would allow for unit testing the conditional logic

File: src/cases/components/case-alert/alert.component.ts [hunk 1]
```diff
@@ -28,8 +28,17 @@ export class AlertComponent implements OnInit, OnDestroy {
     this.routeSubscription = this.router.events.subscribe(() => this.message = '');
     this.alertMessageSubscription = this.alertMessageObservable.subscribe(alert => {
       if (alert) {
-        this.message = alert.message;
-        this.level = alert.level;
+          const msg = alert.message;
+          const caseId = msg.match(/[0-9]{16}/g);
+          if (caseId) {
+
```
Comment (rayliangatos) [inline src/cases/components/case-alert/alert.component.ts:34]: could you remove the space line

File: src/cases/components/case-alert/alert.component.ts [hunk 2]
```diff
@@ -28,8 +28,17 @@ export class AlertComponent implements OnInit, OnDestroy {
...
+            const caseIdHyphen = msg.match(/([0-9][0-9][0-9][0-9])/g).join('-');
+            this.message = msg.replace(caseId.toString(), caseIdHyphen);
+
```
Comment (rayliangatos) [inline src/cases/components/case-alert/alert.component.ts:37]: ditto

File: src/cases/components/case-alert/alert.component.ts [hunk 3]
```diff
@@ -28,8 +28,17 @@ export class AlertComponent implements OnInit, OnDestroy {
     this.routeSubscription = this.router.events.subscribe(() => this.message = '');
     this.alertMessageSubscription = this.alertMessageObservable.subscribe(alert => {
       if (alert) {
-        this.message = alert.message;
-        this.level = alert.level;
+          const msg = alert.message;
```
Comment (rayliangatos) [inline src/cases/components/case-alert/alert.component.ts:31]: should be indented like before

**Review by rayliangatos — COMMENTED** [review]
It needs a unit test to cover the change
currently coverage rate 65%

**Review by rayliangatos — APPROVED** [review]

**Pull request merged**